### ISSUE TITLE
Issues with import to houdini 

### ISSUE THREAD

**Issue by Kingckh** [paulwinex/pw_mGeoExporter#2]
hello , I success to export geo and mtl shader from Maya . I cant see any Mat imported to houdini ,Although geo and Mtl data I can see in file node , can someone help ? thx 

I am using Maya 2017 and houdini 16.671 , thx thx

**Comment by paulwinex** [comment]
Mtl data? Show me screenshots pls. What is the Mtl data?

**Comment by Kingckh** [comment]
Hello master~ I might get it wrong , I would like to ask if the shader can be import to houdini in a easy way ~thx
![untitled](https://user-images.githubusercontent.com/41562250/43067484-11afee56-8e9a-11e8-9542-5c8352832448.png)

**Comment by paulwinex** [comment]
No, shader is not imported. This is only geometry export. In this example exporter make prim groups by maya shader (material). Inside subnet i convert group name to the **shop_materialpath** attribute and assign material name like group name. For example: 
group "lambert1" -> prim attr "/shop/lambert1".
You need just create material manually using maya names.
